application/features/Connection.py

```
@@ -102,6 +115,9 @@
     def _client_connect(self, client: paramiko.SSHClient,
                         host, username,
                         password=None, key_filename=None, private_key_str=None):
+        if self._jump_channel != None:
```

Ruff

Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)